src/dataframe/dataframe.jl

```diff
@@ -1191,6 +1204,13 @@ function disallowmissing!(df::DataFrame, cols::AbstractVector{Bool})
     df
 end
 
+function disallowmissing!(df::DataFrame, cols::Regex)
```

Comment:
`disallowmissing!(df, index(df)[cols])` would be shorter.

Reply:
fixed